homeassistant/components/nest/const.py

```diff
@@ -3,6 +3,10 @@
 DOMAIN = "nest"
 DATA_SDM = "sdm"
 DATA_SUBSCRIBER = "subscriber"
+DATA_NEST_CONFIG = "nest_config"
+
+CONF_PROJECT_ID = "project_id"
+CONF_SUBSCRIBER_ID = "subscriber_id"
```

Comment:
If you are adding these here I think you can remove them from `__init__.py`

Reply:
Oops, fixed.